problems/3.conditionals.js

```diff
@@ -1,30 +1,56 @@
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise
-* @param {number} temp - The current temperature
-* @returns {string} - A message about the temperature
-*/
-function temperatureMessage() {
-
+ * @param {number} temp - The current temperature
+ * @returns {string} - A message about the temperature
+ */
+function temperatureMessage(temp) {
+    let num = -Infinity < 32
+    if (num === temp) {
+        return `It's freezing !`
+    } else {
+        return `It's alright`
+    }
 }
 
+
+
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
-* @param {number} temp - The current temperatur
-* @returns {string} - A message about the temperature
-*/
+ * @param {number} temp - The current temperatur
+ * @returns {string} - A message about the temperature
+ */
 function temperatureMessageUsingTernary() {
```

Comment:
review ternarys. No if else, instead it's a different syntax. 
return temp < 32 ? "cold" : "alright"

Reply:
will do. Thank you